src/components/Carousel/CarouselItem.js

```diff
@@ -14,6 +14,8 @@ const CarouselItem = ({ children }) => {
       {children}
       <style jsx>{`
         .carousel__item {
+          display: flex;
+          flex-direction: column-reverse;
```

Comment:
I've always thought the name should go above the text, but neat solution!

Reply:
Same. But yeah I'm a big fan of flex ordering to preserve the document hierarchy :slightly_smiling_face: